src/Compilers/VisualBasic/Desktop/CommandLine/CommandLineParser.vb

```diff
@@ -1388,6 +1394,19 @@ lVbRuntimePlus:
                           End Function)
         End Function
 
+        Private Function ParseAnalyzerDependencies(name As String, value As String, diagnostics As IList(Of Diagnostic)) As IEnumerable(Of CommandLineAnalyzerDependency)
+            If String.IsNullOrEmpty(value) Then
+                ' TODO: localize <file_list>?
+                AddDiagnostic(diagnostics, ERRID.ERR_ArgumentRequired, name, ":<file_list>")
+                Return SpecializedCollections.EmptyEnumerable(Of CommandLineAnalyzerDependency)()
+            End If
+
+            Return ParseSeparatedPaths(value).
+                   Select(Function(path)
+                              Return New CommandLineAnalyzerDependency(path)
+                          End Function)
```

Comment:
Use single line lambda :) i.e. Return ParseSeperatedPaths(value).Select(Function(path) New CommandLineAnalyzerDependency(path)

or better yet consider using LINQ syntax as VB user's do more often :) i.e. Return From path in ParseSeparatedPaths(value) Select New CommandLineAnalyzerDependency(path)